lib/bdev/nvme/blockdev_nvme.c

```diff
@@ -763,11 +809,36 @@ bdev_nvme_library_init(void)
 		probe_ctx.count++;
 	}
 
-	g_reset_controller_on_timeout =
-		spdk_conf_section_get_boolval(sp, "ResetControllerOnTimeout", false);
+	if ((g_timeout = spdk_conf_section_get_intval(sp, "Timeout")) < 0) {
+		/* Check old name for backward compatibility */
+		if ((g_timeout = spdk_conf_section_get_intval(sp, "NvmeTimeoutValue")) < 0) {
+			g_timeout = 0;
+		} else {
+			SPDK_WARNLOG("NvmeTimeoutValue was renamed to Timeout\n");
+			SPDK_WARNLOG("Please update your configuration file\n");
+		}
+	}
 
-	if ((g_timeout = spdk_conf_section_get_intval(sp, "NvmeTimeoutValue")) < 0) {
-		g_timeout = 0;
+	if (g_timeout > 0) {
```

Comment:
I think it would be a mistake for us to provide a default timeout value because we can't predict what a reasonable value is for all possible NVMe devices. By default, the code won't time any I/O out. If the user chooses to enable it in their config file, then the user should be willing to pick a value for the timeout based on the current hardware.